atd-toolbox/get_lookup_table_changes/README.md

```diff
@@ -1,27 +1,72 @@
-This folder and the script it contains is for the purpose of getting our lookup tables up to date with the latest lookup export from CRIS. We will probably need to run this after every major CRIS release.
+This helper is for the purpose of getting our lookup tables up to date with the latest lookup export from CRIS. It generates up and down migrations which can be applied to the database in order to bring our database back in sync with the lookup values that CRIS provides in extract files.
+
+This tool should be run after every major CRIS release.
 
 ### Install packages
 
 Create or activate your virtual python environment using [venv](https://docs.python.org/3/library/venv.html).
 
 Install packages from the requirements.txt file:
 
-`pip install -r requirements.txt`
+```shell
+$ pip install -r requirements.txt
```

Comment:
I set up a fresh venv when testing, and I noticed that `requests` is missing from the requirements file.

Reply:
:pray:

Reply:
thanks for catching this, Mike!